tests/perf/main/install-many-snaps-no-wait/task.yaml

```
@@ -0,0 +1,59 @@
+summary: Ensure the system properly handles installing many snaps with the --no-wait flag
+
+details: |
+    Install different snaps many times based on arch availability. In this
+    scenario all the snaps are installed in parallel and the test checks
+    the system can handel all the installs and ramains running.
```

I missed this one earlier
```suggestion
    the system can handle all the installs and remains running.
```